gradlew

```diff
@@ -1,17 +1,5 @@
 #!/usr/bin/env sh
 
-#
-# Copyright OpenSearch Contributors
-# SPDX-License-Identifier: Apache-2.0
-#
-# The OpenSearch Contributors require contributions made to
-# this file be licensed under the Apache-2.0 license or a
-# compatible open source license.
-#
-# Modifications Copyright OpenSearch Contributors. See
-# GitHub history for details.
```

Comment:
could you keep license header, same for gradlew.bat

Reply:
Will do it. I created gradle wrapper for 7 and it replaced the license header.